store/sqlstore/emoji_store.go

```diff
@@ -122,25 +122,22 @@ func (es SqlEmojiStore) GetByName(name string) store.StoreChannel {
 	})
 }
 
-func (es SqlEmojiStore) GetMultipleByName(names []string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		keys, params := MapStringsToQueryParams(names, "Emoji")
+func (es SqlEmojiStore) GetMultipleByName(names []string) ([]*model.Emoji, *model.AppError) {
+	keys, params := MapStringsToQueryParams(names, "Emoji")
 
-		var emojis []*model.Emoji
+	var emojis []*model.Emoji
 
-		if _, err := es.GetReplica().Select(&emojis,
-			`SELECT
+	if _, err := es.GetReplica().Select(&emojis,
+		`SELECT
 				*
```

Comment:
Please keep the indentation consistent here.

Reply:
Done.